Trappist/src/Promact.Trappist.Repository/ProfileDetails/ProfileRepository.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Promact.Trappist.DomainModel.DbContext;
+using Promact.Trappist.Web.Models;
+
+namespace Promact.Trappist.Repository.ProfileDetails
+{
+    public class ProfileRepository :IProfileRepository
+    {
+    private readonly TrappistDbContext _dbContext;
+
+    public ProfileRepository(TrappistDbContext dbContext)
+    {
+      _dbContext = dbContext;
+    }
+
+    /// <summary>
+    /// Updates current user's details
+    /// </summary>
+    /// <param name="updateUserDetails"></param>
+    /// <returns>Update user details in database</returns>
+    public void UpdateProfile(ApplicationUser updateUserDetails)
+    {
+      var user = _dbContext.Users.Update(updateUserDetails);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused 'user' local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1481)